src/gcp_scanner/crawl.py

```diff
@@ -281,8 +280,8 @@ def get_subnets(project_name: str,
     while request is not None:
       response = request.execute()
       if response.get("items", None) is not None:
-        for name, subnetworks_scoped_list in response["items"].items():
-          subnets_list.append((name, subnetworks_scoped_list))
+        subnets_list = [(name, subnetworks_scoped_list) 
+                        for name, subnetworks_scoped_list in response["items"].items()]
```

Comment:
This is redundant. Replace with just `subnets_list = response["items"].items()`

Reply:
Hi @ZetaTwo, I have made the requested changes in the latest PR. Check and let me know if any changes required.

Reply:
This is still not fixed